packages/walletconnect/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/walletconnect",
-  "version": "2.1.3",
+  "version": "2.2.0",
```

```suggestion
  "version": "2.2.0-alpha.1",
```
Alpha tag for pre-publish release